src/Chameleon.php

```diff
@@ -94,16 +94,6 @@ public static function init() {
 		}
 	}
 
-	/**
-	 * @param OutputPage $out
-	 */
-	public function initPage( OutputPage $out ) {
-		parent::initPage( $out );
-
-		// Enable responsive behaviour on mobile browsers
-		$out->addMeta( 'viewport', 'width=device-width, initial-scale=1, shrink-to-fit=no' );
```

Comment:
This is based on Bootstrap 4's recommended meta tag:
https://getbootstrap.com/docs/4.6/getting-started/introduction/#responsive-meta-tag

Can you confirm that the difference between that and MediaWiki's tag does not cause issues or changes in mobile behavior?

```
..., shrink-to-fit=no"/>
..., user-scalable=yes, minimum-scale=0.25, maximum-scale=5.0"/>
```